src/dotnet/APIView/APIViewWeb/Languages/CSharpLanguageService.cs

```diff
@@ -11,17 +12,61 @@ namespace APIViewWeb
     public class CSharpLanguageService : LanguageService
     {
         public override string Name { get; } = "C#";
-
         public override string Extension { get; } = ".dll";
 
+        public override bool IsSupportedFile(string name)
+        {
+            return IsDll(name) || IsNuget(name);
+        }
+
+        private static bool IsDll(string name)
```

Comment:
Should we enforce nupkg uploads?

Reply:
Might be rude. Let's give people time to adjust.